roles/openshift_aws/tasks/provision.yml

```diff
@@ -34,14 +34,14 @@
   include: elb.yml
   vars:
     openshift_aws_elb_direction: internal
-    openshift_aws_elb_name: "{{ openshift_aws_clusterid }}-{{openshift_aws_node_group_type }}-internal"
+    l_openshift_aws_elb_name: "{{ openshift_aws_elb_name }}-internal"
```

Comment:
Why the change here?

Reply:
This is overriding the inventory variable openshift_aws_elb_name.

If this were to be set with extra_vars, it would force-drop -internal and -external.

If someone wants to create and call elb directly, they could set l_openshift_aws_elb_name in inventory.

Reply:
I see what you mean. We should probably test to see if it's set and take that. If not, continue using the logic for internal and external. I'd prefer that if someone calls elb directly they use the openshift variable rather than the l_ version.

Reply:
I agree.  That would be good enhancement.  For me, this is a bug fix until we can sanitize clusterid.